mula/scheduler/schedulers/normalizer.py

```diff
@@ -161,41 +173,46 @@ def push_task(self, normalizer: Plugin, raw_data: RawData, caller: str = "") ->
 
         if not self.is_task_allowed_to_run(normalizer):
             self.logger.debug(
-                "Task is not allowed to run: %s [organisation_id=%s, scheduler_id=%s, caller=%s]",
-                task,
-                self.organisation.id,
-                self.scheduler_id,
-                caller,
+                "Task is not allowed to run: %s",
+                task.id,
+                task_id=task.id,
+                organisation_id=self.organisation.id,
+                scheduler_id=self.scheduler_id,
+                caller=caller,
             )
             return
 
         try:
             if self.is_task_running(task):
                 self.logger.debug(
-                    "Task is already running: %s [organisation_id=%s, scheduler_id=%s, caller=%s]",
-                    task,
-                    self.organisation.id,
-                    self.scheduler_id,
-                    caller,
+                    "Task is still running: %s",
+                    task.id,
+                    task_id=task.id,
+                    organisation_id=self.organisation.id,
+                    scheduler_id=self.scheduler_id,
+                    caller=caller,
                 )
                 return
-        except Exception:
+        except Exception as exc_running:
```

Comment:
`exc_running` is not needed; passing `exc_info=True` to `logger.warning` will use the last exception